### PR TITLE
Reorder some f32/f64 ops

### LINKED ISSUE #185: Float opcode ordering
OpCodes for float math were changed shortly after the initial reordering.
https://github.com/WebAssembly/design/pull/828/files

I started the pull together a PR to fix that, but I don't know how to easily fix all the baselines that depends on the opcodes' value

### PR DESCRIPTION
**Pull request by binji**
Fixes #185.

### CODE REVIEW

File: src/wasm-common.h [hunk 1]
```diff
@@ -298,32 +298,32 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
...
-  V(F32, F32, F32, 0, 0x90, F32_TRUNC, "f32.trunc")                     \
-  V(F32, F32, F32, 0, 0x91, F32_NEAREST, "f32.nearest")                 \
-  V(F32, F32, F32, 0, 0x92, F32_SQRT, "f32.sqrt")                       \
-  V(F32, F32, F32, 0, 0x93, F32_ADD, "f32.add")                         \
-  V(F32, F32, F32, 0, 0x94, F32_SUB, "f32.sub")                         \
-  V(F32, F32, F32, 0, 0x95, F32_MUL, "f32.mul")                         \
-  V(F32, F32, F32, 0, 0x96, F32_DIV, "f32.div")                         \
-  V(F32, F32, F32, 0, 0x97, F32_MIN, "f32.min")                         \
-  V(F32, F32, F32, 0, 0x98, F32_MAX, "f32.max")                         \
+  V(F32, F32, F32, 0, 0x8d, F32_CEIL, "f32.ceil")                       \
```
Comment (KarlSchimpf) [inline src/wasm-common.h]: These numbers are not matching what is in
[BinaryEncoding.md](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).

According to that:
f32.ceil  is 0x7e
f32.floor is 0x7f
f32.trunc is 0x80
etc.

File: src/wasm-common.h [hunk 2]
```diff
@@ -298,32 +298,32 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
...
-  V(F64, F64, F64, 0, 0x9e, F64_TRUNC, "f64.trunc")                     \
-  V(F64, F64, F64, 0, 0x9f, F64_NEAREST, "f64.nearest")                 \
-  V(F64, F64, F64, 0, 0xa0, F64_SQRT, "f64.sqrt")                       \
-  V(F64, F64, F64, 0, 0xa1, F64_ADD, "f64.add")                         \
-  V(F64, F64, F64, 0, 0xa2, F64_SUB, "f64.sub")                         \
-  V(F64, F64, F64, 0, 0xa3, F64_MUL, "f64.mul")                         \
-  V(F64, F64, F64, 0, 0xa4, F64_DIV, "f64.div")                         \
-  V(F64, F64, F64, 0, 0xa5, F64_MIN, "f64.min")                         \
-  V(F64, F64, F64, 0, 0xa6, F64_MAX, "f64.max")                         \
+  V(F64, F64, F64, 0, 0x9b, F64_CEIL, "f64.ceil")                       \
```
Comment (KarlSchimpf) [inline src/wasm-common.h]: Similarly, f64.ceil is 0x92

File: src/wasm-common.h (hunk 1 above)
Comment (KarlSchimpf) [inline src/wasm-common.h]: Never mind. I forgot to look in branch for 0xd.

**Comment by Cellule** [comment]
Thanks a lot for the quick fix

File: src/wasm-common.h (hunk 1 above)
Comment (binji) [inline src/wasm-common.h]: Take a look at the doc for binary 0xd: https://github.com/WebAssembly/design/blob/binary_0xd/BinaryEncoding.md

**Pull request merged**